smf/zone-network-setup/manifest.xml

```diff
@@ -31,6 +31,10 @@
     timeout_seconds='0' />
 
   <exec_method type='method' name='stop' exec=':true' timeout_seconds='0' />
+  <!-- We use the same command as the start method as it is safe to rerun. -->
+  <exec_method type='method' name='refresh'
+    exec='/opt/oxide/zone-setup-cli/bin/zone-setup common-networking -d %{config/datalink} -s %{config/static_addr} -g %{config/gateway}'
+    timeout_seconds='0' />
```

Comment:
What happens if the service is currently running when it's refreshed?

Reply:
Nothing really. All of the commands on that service are idempotent (I also manually refreshed this service multiple times to see what happened).

Additionally, it's highly unlikely this would happen. Several other services on the switch zone depend on this service, which means we wouldn't even get to the point where the service is refreshed, if the zone-network-setup service hadn't exited with 0